ui/app/components/app/modal/modal.component.js

```diff
@@ -41,10 +43,12 @@ export default class Modal extends PureComponent {
       cancelText,
       contentClass,
       containerClass,
+      hideFooter,
+      className = '',
     } = this.props
 
     return (
-      <div className={classnames('modal-container', containerClass)}>
+      <div className={classnames('modal-container', className, containerClass)}>
```

Comment:
- [ ] the addition of this `className` prop is likely unnecessary given the `containerClass` prop